docs/MultiStepDonationForm-and-FormControllers.md

```diff
@@ -0,0 +1,88 @@
+# Multi Step Donation Form and Form Controllers
+
+Since the 2022 campaign a lot of tests have been focused on multistep forms, each with different flows. These tests are different enough functionally that they would require a new `DonationForm` component for each.
+
+In order to avoid duplicating components per-test we opted to instead create a flexible `MultiStepDonationForm` that is provided with different implementations of a `FormController` which contains the logic to run the user path on the form.
```

Comment:
```suggestion
In order to avoid duplicating components per-test we opted to instead create a flexible `MultiStepDonationForm` that is provided with different implementations of a `FormController` which contains the logic that determines which form page to show, based on the user interaction with the current form page.
```